releasenotes/notes/update-pulse-gate-pass-for-target-ebfb0ec9571f058e.yaml

```diff
@@ -0,0 +1,23 @@
+---
+features:
+  - |
+    A new method :meth:`.CalibrationEntry.get_duration` has been added to
+    calibration entries. This method returns duration of calibration
+    in units of dt when available, otherwise returns None.
+upgrade:
+  - |
+    :meth:`.Target.update_from_instruction_schedule_map` no longer raises
+    KeyError nor ValueError when qubits are missing in the target instruction
+    or inst_name_map is not provided for undefined instruction.
+    In the former case, it just ignores the inst map definition for undefined qubits.
+    In the latter case, gate mapping is pulled from the standard Qiskit gates
+    and finally custom :class:`.Gate` object is defined from the schedule name.
+  - |
+    :class:`PulseGates` transpiler pass has been upgraded to respect
```

Comment:
This and the bug fix note don't need a release note because this functionality was never release, so we're not changing anything.

Reply:
Is adding this to milestone 0.24 the right way to capture that ideally this would merged before the next release so that the bug is never in a release?

Reply:
It is and marking it as high priority which indicates it is release blocking

Reply:
Removed in [1f8404f](https://github.com/Qiskit/qiskit-terra/pull/9587/commits/1f8404f3c1209a106651f2028f871ffa9dffcb61)